src/Http/Http.Abstractions/src/Resources.resx

```diff
@@ -162,4 +162,10 @@
   <data name="RouteValueDictionary_DuplicatePropertyName" xml:space="preserve">
     <value>The type '{0}' defines properties '{1}' and '{2}' which differ only by casing. This is not supported by {3} which uses case-insensitive comparisons.</value>
   </data>
+  <data name="Exception_KeyedServicesNotSupported" xml:space="preserve">
+    <value>This service provider doesn't support keyed services.</value>
+  </data>
+  <data name="Exception_NoServiceRegistered" xml:space="preserve">
```

Comment:
Sorry, there is this one too 😢

Reply:
Hehe, don't sweat it, should've checked it. Will run by all resources 👍